addon/TabTracker.js

```diff
@@ -177,7 +178,7 @@ TabTracker.prototype = {
       return;
     }
     if (this._tabData.start_time) {
-      this._tabData.session_duration = (Date.now() - this._tabData.start_time);
+      this._tabData.session_duration = (absPerf.now() - this._tabData.start_time);
```

Comment:
nit: according to [here](https://github.com/mozilla/activity-stream/blob/master/data_dictionary.md), `session_duration` should be an integer in milliseconds, let's round up the result of performance.now here